ferrocene/doc/user-manual/src/conf.py

```diff
@@ -12,13 +12,11 @@
     "ferrocene_domain_cli",
     "ferrocene_autoglossary",
 
+    "myst_parser",
+
     "sphinx.ext.intersphinx",
-    "sphinx.ext.autosectionlabel",
```

Comment:
With `sphinx.ext.autosectionlabel` it fails with:
> /home/urhengulas/Documents/github.com/ferrocene/ferrocene/src/tools/rustfmt/Configurations.md:133: WARNING: duplicate label rustfmt/config:example, other instance in /home/urhengulas/Documents/github.com/ferrocene/ferrocene/ferrocene/doc/user-manual/src/rustfmt/config.rst
> /home/urhengulas/Documents/github.com/ferrocene/ferrocene/src/tools/rustfmt/Configurations.md:253: WARNING: duplicate label rustfmt/config:`"samelinewhere"` (default):, other instance in /home/urhengulas/Documents/github.com/ferrocene/ferrocene/ferrocene/doc/user-manual/src/rustfmt/config.rst
> /home/urhengulas/Documents/github.com/ferrocene/ferrocene/src/tools/rustfmt/Configurations.md:268: WARNING: duplicate label rustfmt/config:`"alwaysnextline"`:, other instance in /home/urhengulas/Documents/github.com/ferrocene/ferrocene/ferrocene/doc/user-manual/src/rustfmt/config.rst

Can we remove it?

Reply:
Yeah.